### ISSUE TITLE
Error: The option curriculum was specified in your YAML file, but is invalid.

### ISSUE THREAD

**Issue by nntsrb** [Unity-Technologies/ml-agents#4439]
I have this error when I tried to train with the file below. It used to work before, but now it seems there are some problem with the curriculum file.

```
behaviors:
  TileBaseISO:
    trainer_type: ppo
    hyperparameters:
      batch_size: 2048
      buffer_size: 20480
      learning_rate: 0.0003
      beta: 0.005
      epsilon: 0.2
      lambd: 0.95
      num_epoch: 3
      learning_rate_schedule: constant
    network_settings:
      normalize: false
      hidden_units: 512
      num_layers: 2
      vis_encode_type: simple
    reward_signals:
      extrinsic:
        gamma: 0.99
        strength: 1.0
    keep_checkpoints: 5
    max_steps: 50000000
    time_horizon: 1000
    summary_freq: 10000
    threaded: true
    self_play:
      save_steps: 50000
      team_change: 200000
      swap_steps: 2000
      window: 10
      play_against_latest_model_ratio: 0.5
      initial_elo: 1200.0

curriculum:
  TileBaseISO:
    measure: reward
    thresholds: [-1.8, -1.2, -0.5, 0.5, 1.2, 1.8]
    min_lesson_length: 80
    signal_smoothing: true
    parameters:
      mosquito_count: [8.0, 7.0, 6.0, 5.0, 4.0, 3.0]
```

**Comment by harperj** [comment]
Hi @nntsrb -- the config format for curriculum learning did in face change.  We've updated the docs on how to configure curriculum learning here: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-ML-Agents.md#curriculum

**Comment by jwson97** [comment]
I also have same problem. But I change my yaml file like this : 
but when I check "checkpoint_radius" parameter value in tensorboard after training. The value is 0 which is set as default.
curriculum is not working. What is the problem.

behaviors:
    AircraftPlayer:
        trainer_type: ppo 
        hyperparameters: 
            batch_size : 2048
            beta: 1.0e-2
            buffer_size: 20480
            epsilon: 0.2
            lambd: 0.95
            learning_rate: 3.0e-4
            learning_rate_schedule: linear
            num_epoch: 3
        network_settings: 
            hidden_units: 256
            memory:
                memory_size : 256
                sequence_length: 64
            normalize : false
            num_layers: 2
            vis_encoder_type: simple
        reward_signals: 
            extrinsic:
                strength: 1.0
                gamma: 0.99
        max_steps: 5.0e6
        time_horizon: 128
        summary_freq: 32000
        threaded: true
        
#curriculum:
#    AircraftPlayer:
#        mesaure : reward
#        thresholds: [2.0, 20.0, 20.0, 4.0, 6.0]
#        min_lesson_length : 200
#        parameters : 
#            checkpoint_radius: [70.0, 50.0, 30.0, 20.0, 10.0, 0.0]
        
    
environment_parameters:
    checkpoint_radius: 
        curriculum: 
          - name : First
            completion_criteria:
                measure : reward
                min_lesson_length : 200
                threshold : 2.0
                signal_smoothing : true
            value : 70.0
          - name : Second
            completion_criteria:
                measure : reward
                min_lesson_length : 200
                threshold : 2.0
                signal_smoothing : true
            value : 50.0
          - name : Third
            completion_criteria:
                measure : reward
                min_lesson_length : 200
                threshold : 2.0
                signal_smoothing : true
            value : 30.0
          - name : Forth
            completion_criteria:
                measure : reward
                min_lesson_length : 200
                threshold : 4.0
                signal_smoothing : true
            value : 20.0
          - name : Fifth
            completion_criteria:
                measure : reward
                min_lesson_length : 200
                threshold : 6.0
                signal_smoothing : true
            value : 10.0
          - name : Last
            value : 0.0
